### PR TITLE
feat: add create/edge-storage/object command

### PR DESCRIPTION
**Pull request by maxwelbm**
Edge storage objects create command

### CODE REVIEW

File: utils/helpers.go [hunk 1]
```diff
@@ -279,6 +279,9 @@ func WriteAzionJsonContent(conf *contracts.AzionApplicationOptions) error {
 
 // Returns the correct error message for each HTTP Status code
 func ErrorPerStatusCode(httpResp *http.Response, err error) error {
+	//if httpResp != nil {
```
Comment (PabloDiehl) [inline utils/helpers.go:282]: Do we need these commented lines? 🤔

**Review by PabloDiehl — APPROVED** [review]

File: utils/helpers.go (hunk 1 above)
Comment (maxwelbm) [reply utils/helpers.go:282]: At the moment it's not an idea, to minimize code but it affects many tests, so I'll need to send it in a separate pr

**Review by PabloDiehl — APPROVED** [review]

File: messages/edge_storage/messages.go [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
 
-	SHORT_DESCRIPTION               = "Manages Edge Storage buckets and objects directly through the command line"
-	SHORT_DESCRIPTION_CREATE_BUCKET = "Creates a bucket in Edge Storage"
-	SHORT_DESCRIPTION_LIST_BUCKET   = "List the buckets in Edge Storage"
-	SHORT_DESCRIPTION_DELETE_BUCKET = "Deleted the bucket in Edge Storage"
+	SHORT_DESCRIPTION                = "Manages Edge Storage buckets and objects directly through the command line"
+	SHORT_DESCRIPTION_CREATE_BUCKET  = "Creates a bucket in Edge Storage"
+	SHORT_DESCRIPTION_LIST_BUCKET    = "List the buckets in Edge Storage"
+	SHORT_DESCRIPTION_DELETE_BUCKET  = "Deleted the bucket in Edge Storage"
+	SHORT_DESCRIPTION_CREATE_OBJECTS = "Creates a objects in Edge Storage"
```
Comment (PatrickMenoti) [inline messages/edge_storage/messages.go:12]: Creates objects in Edge Storage

File: messages/edge_storage/messages.go [hunk 3]
```diff
@@ -1,26 +1,31 @@
...
 
-	LONG_DESCRIPTION               = "Allows users to perform a wide range of operations, from creating and deleting buckets to adding, removing, and manipulating objects."
-	LONG_DESCRIPTION_CREATE_BUCKET = "Allows users to create a bucket in Edge Storage"
-	LONG_DESCRIPTION_LIST_BUCKET   = "Allows users to list their buckets in Edge Storage"
-	LONG_DESCRIPTION_DELETE_BUCKET = "Allows users to delete their buckets in Edge Storage"
+	LONG_DESCRIPTION                = "Allows users to perform a wide range of operations, from creating and deleting buckets to adding, removing, and manipulating objects."
+	LONG_DESCRIPTION_CREATE_BUCKET  = "Allows users to create a bucket in Edge Storage"
+	LONG_DESCRIPTION_LIST_BUCKET    = "Allows users to list their buckets in Edge Storage"
+	LONG_DESCRIPTION_DELETE_BUCKET  = "Allows users to delete their buckets in Edge Storage"
+	LONG_DESCRIPTION_CREATE_OBJECTS = "Allows users to create a objects in Edge Storage"
```
Comment (PatrickMenoti) [inline messages/edge_storage/messages.go:18]: Allows users to create objects in Edge Storage

File: messages/edge_storage/messages.go [hunk 4]
```diff
@@ -29,13 +34,20 @@ const (
...
+	FLAG_NAME_CREATE_SOURCE        = "Path to the local file that will be uploaded to the bucket, path absoluto"
+	FLAG_FILE_JSON_CREATE_OBJECTS  = "Path to a JSON file containing the attributes of the objects that will be created; you can use - for reading from stdin"
+	FLAG_HELP_CREATE_OBJECTS       = "Displays more information about the create edge-storege objects command"
 
 	ASK_NAME_CREATE_BUCKET         = "Enter your bucket's name"
 	ASK_EDGE_ACCESSS_CREATE_BUCKET = "Enter your bucket's edge access type"
 	ASK_NAME_DELETE_BUCKET         = "Enter the name of the Bucket you wish to delete"
 	ASK_NAME_UPDATE_BUCKET         = "Enter the name of the Bucket you wish to update"
+	ASK_OBJECT_KEY_CREATE_OBJECT   = "Enter your object name"
+	ASK_SOURCE_CREATE_OBJECT       = "Enter your path source"
```
Comment (PatrickMenoti) [inline messages/edge_storage/messages.go:47]: Enter your source path

**Review by PabloDiehl — DISMISSED** [review]

**Review by PatrickMenoti — APPROVED** [review]

**Comment by maxwelbm** [comment]
@PatrickMenoti 💟

**Review by PabloDiehl — APPROVED** [review]

**Pull request merged**